### DISCUSSION TITLE
Add existing catalog item to listing with POST_PRODUCT_DATA XML feed

### DISCUSSION THREAD

**Discussion by njiang** [amzn/selling-partner-api-models#3742] [General] [answered]
I'm following this link to create inbound shipment. https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v0-use-case-guide#step-2-create-a-listing-and-convert-it-to-fba

Step 2 is to add existing catalog items to listing so they can be converted to FBA.  I'm trying to call the feed api to submit an XML POST_PRODUCT_DATA feed.  I tried something like this but ketp receiving error like this:

 {"errors":[{"code":"InvalidInput","message":"Invalid request parameters","details":""}]}

I'm wondering if I need to include product data in the feed?  Or is there any information missing?  

```
<?xml version="1.0" encoding="UTF-8"?>
<AmazonEnvelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="amzn-envelope.xsd">
    <Header>
        <DocumentVersion>1.01</DocumentVersion>
        <MercharntIdentifier>{seller id}</MercharntIdentifier>
    </Header>
    <MessageType>Product</MessageType>
    <Message>
        <MessageID>listing-4-0</MessageID>
        <OperationType>UPDATE</OperationType>
        <Product>
            <SKU>test-sku-3</SKU>
            <StandardProductID>
                <Type>ASIN</Type>
                <Value>{asin}</Value>
            </StandardProductID>
            <DescriptionData>
                <Title>test product 13</Title>
                <Description>this is a test product 11</Description>
            </DescriptionData>
        </Product>
    </Message>
</AmazonEnvelope>

```

**Comment by littleylv** [comment]
> <MessageID>listing-4-0</MessageID>

MessageID must be positive integer.

**Reply by njiang** [reply]
Thanks for the response. I changed message ID to 1, but still the same error, without any info on what's wrong.  Do I have to include ProductData section?  I'm trying to list an existing catalog item and I don't think it's necessary? Is there any sample XML feed for that purpose?

**Reply by littleylv** [reply, marked as answer]
from the url you post:

> If the product is in Amazon's catalog, the feed should include:

> SKU, containing the seller's unique SKU for the product.

> StandardProductID, containing values that uniquely identify the product in Amazon's catalog.

> **Battery and dangerous goods information**, according to the guidance in [Requirements for MWS Integrators (PDF)](https://images-na.ssl-images-amazon.com/images/G/01/rainier/help/xsd/release_4_1/MWSDangerousGoodsAttributesConditionalRequirementsOct182017.pdf).

so try this:

```
<DescriptionData>
  <Title>test product 13</Title>
  <Description>this is a test product 11</Description>
  <Battery>
    <AreBatteriesIncluded>false</AreBatteriesIncluded>
    <AreBatteriesRequired>false</AreBatteriesRequired>
  </Battery>
  <SupplierDeclaredDGHZRegulation>not_applicable</SupplierDeclaredDGHZRegulation>
</DescriptionData>`
